WBooks/Library/LibraryViewController.swift

```diff
@@ -21,12 +21,24 @@ class LibraryViewController: UIViewController {
         super.viewDidLoad()
         libraryView.tableView.delegate = self
         libraryView.tableView.dataSource = self
+        navigationController?.navigationBar.isHidden = false
+        navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(named: "notification"), style: .plain, target: self, action: #selector(notificationPress))
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "search"), style: .plain, target: self, action: #selector(searchPress))
```

Comment:
remember to add the images to the UIImage extension

Reply:
Done